---
layout: default
---

## WABI 2025 Poster presentations

Poster presenters, please place your posters on the board marked by the number listed here for your poster.

Make the posters at most 4 ft wide and 3 ft high.

<table><thead>
<tr><th>Poster number</th><th>Authors</th><th>Title</th></tr>
</thead><tbody>
<tr><td>1</td><td>Rossie S. Luo, Ellen E. Vaughan, Justin R. Clark, Anthony W. Maresso, Michael G. Nute and Todd J. Treangen</td><td>Strainify: Strain-Level Abundance Estimation for Short-read Metagenomics</td></tr>
  <tr><td>2</td><td>Steven Tan, Sina Majidian, Ben Langmead and Mohsen Zakeri</td><td>Movi Color: fast and accurate long-read classification with the move structure</td></tr>
  <tr><td>3</td><td>Peter Heringer and Daniel Doerr</td><td>Efficient data structures for coverage-based pangenome statistics</td></tr>
  <tr><td>4</td><td class="s0" dir="ltr">Sina Majidian and Ben Langmead</td><td>MetaKpick: machine learning–based metagenomic classification with multi k-mer-based pangenome indexes information on submission</td></tr>
  <tr><td>5</td><td>Ryan Doughty, Michael Tisza and Todd Treangen</td><td>bronko: ultra-rapid mapping-free detection of intrahost variation from viral amplicon sequencing data</td></tr>
  <tr><td>6</td><td>Anning Cui, Mihai Pop and Manasa Annavaram</td><td>Exploring structural variants in microbiome communities through simplified graph representations</td></tr>
  <tr><td>7</td><td>Michael X. Wang, Bryce Kille, Michael G. Nute and Todd J. Treangen</td><td>Seqwin: Identifying strain-specific signature sequences via minimizer graphs in 10s of thousands of microbial genomes</td></tr>
  <tr><td>8</td><td>Mao-Jan Lin, Ben Langmead and Yana Safonova </td><td>Extending IGLoo to light chains: High-resolution analysis and assembly of IGK and IGL loci in lymphoblastoid cell lines with PacBio HiFi reads</td></tr>
  <tr><td>9</td><td>Md Hasin Abrar, Paul Medvedev and Giorgio Vinciguerra</td><td>Efficiency of Learned Indexes on Genome Spectra</td></tr>
  <tr><td>10</td><td>Nathaniel K. Brown, Lore Depuydt, Mohsen Zakeri, Anas Alhadi, Nour Allam, Dove Degleiter, Nithin Bharathi, Suchith Sridhar, Hamza Wahed, Travis Gagie and Benjamin Langmead</td><td>KeBaB: k-mer based breaking for finding long MEMs</td></tr>
  <tr><td>11</td><td>Haonan Wu, Antonio Blanca and Paul Medvedev</td><td>A k-mer-based estimator of the substitution rate between repetitive sequences</td></tr>
  <tr><td>12</td><td>Rachel Parsons, Jessica M. Storer and Erin K. Molloy</td><td>A novel pipeline for calling transposable elements presence/absence in repetitive regions from whole genome alignments</td></tr>
  <tr><td>13</td><td>Elizabeth Hunter and Rob Patro</td><td>Quantifying uncertainty in metagenomic analysis: the impact of phylogenetic representation on classification entropy with Kraken2</td></tr>
  <tr><td>14</td><td>Nakyung Lee and Mihai Pop</td><td>Assessing the Impact of Database Quality on the Performance of Alignment-Based Sequence Search Tools</td></tr>
  <tr><td>15</td><td>Sina Majidian, Stephen Hwang, Mohsen Zakeri and Ben Langmead</td><td>EvANI benchmarking workflow for evolutionary distance estimation</td></tr>
  <tr><td>16</td><td>Yunzhuo Liu, Wesley Demontigny, Seyed Mohammad Saleh Ghaemi, Stratis Aloimonos, Jenan El-Hifnawi and Erin K. Molloy</td><td>Evaluating the impact of species tree shape on new and old methods for species tree estimation</td></tr>
  <tr><td>17</td><td>Parth Dua, Rachel Parsons, Yunzhuo Liu and Erin K. Molloy</td><td>Evaluating gene tree-based methods for orthology inference and their impact on species tree estimation</td></tr>
  <tr><td>18</td><td>Luiz Mata Lopez, Junyan Dai and Erin K. Molloy</td><td>Evaluating tumor phylogeny methods under single nucleotide and copy number changes</td></tr>
  <tr><td>19</td><td>Nathalie Bonin, Rain Haworth and Mihai Pop</td><td>An Evaluation of DeepARG – Does Deep Learning Really Provide a Significant Improvement Over the Best Hit Approach?</td></tr>
  <tr><td>20</td><td>Zahra Zare Jousheghani, Noor Pratap Singh and Rob Patro</td><td>Oarfish improves long-read RNA-seq quantification through enhanced probabilistic modeling of transcript abundance</td></tr>
  <tr><td>21</td><td>Rain Haworth, Seth Commichaux and Mihai Pop</td><td>Towards adversarial robustness in biological sequence databases: identifying random mutations in 16S rRNA</td></tr>
  <tr><td>22</td><td>Zhezhen Wang and Rob Patro</td><td>Spatial Data Structures for Representing and Compressing Spatial Transcriptomics data</td></tr>
  <tr><td>23</td><td>Bunsho Koyano and Tetsuo Shibuya</td><td>Fast and Accurate Comparison of Protein Conformational Ensembles</td></tr>
  </tbody></table>
